matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -160,26 +160,32 @@ async fn message_loop<M: Messenger>(
 
         select! {
             _  = &mut timeout => {
-                requests_sender.unbounded_send(PeerRequest::KeepAlive).expect("send failed");
-                // UNWRAP: we will only ever get here if there already was a timeout
-                let interval = keep_alive_interval.unwrap();
-                timeout = Either::Left(Delay::new(interval)).fuse();
+                // We don't care if a KeepAlive fails to send
```

Comment:
do we really not care about this? as far as I can tell, it will happen if the signalling loop is dropped (or exits). Wouldn't it be natural for the message loop to assume that signaling is functioning, or at least warn if it isn't?

Reply:
a warn is good. I figured if the loop was dropped or exits it would fail on the next send attempt, which seemed better.

Reply:
actually I take it back, I'll treat it as fatal.